### PR TITLE
Spectral type updates

### PR DESCRIPTION
**Pull request by mdecleir**
I have updated some spectral types and the references to the spectral types, to be consistent with what will be in the NIR extinction paper.
I also added masks for some stars to exclude bad wavelength regions in the SpeX spectra.

### CODE REVIEW

File: DAT_files/hd183143.dat [hunk 1]
```diff
@@ -13,7 +13,8 @@ WISE1 = 3.307 +/- 0.478 ; WISE ref = IRSA AllWISE Source Catalog
 WISE2 = 2.817 +/- 0.483
 WISE3 = 3.053 +/- 0.012
 WISE4 = 2.768 +/- 0.021
-sptype = B7Iae ; sptype ref = Hobbs+2009 (2009ApJ...705...32H)
+sptype = B7Ia ; sptype ref = Morgan, Code & Whitford 1955 (1955ApJS....2...41M)
 corfac_spex_SXD = 0.593
 LXD_man = True
 corfac_spex_LXD = 0.62
+#spex_mask = [(1.347,1.411),(1.801,1.942),(2.546,2.86),(4.176,4.583)]
```
Comment (karllark) [inline DAT_files/hd183143.dat:20]: Commented out.  Should it be included then?

**Review by karllark — CHANGES_REQUESTED** [review]
Looks good except for the commented out lines?  Worth keeping or should they be removed.  Probably other commented out lines in other files.  Something we should maybe talk about briefly and decide what to do with them.

File: DAT_files/hd183143.dat (hunk 1 above)
Comment (mdecleir) [reply DAT_files/hd183143.dat]: Nope, this was a leftover from a test. I removed it now.

Comment (karllark) [reply DAT_files/hd183143.dat]: There other files with `#spex_mask` lines.  Maybe you could check those as well?  I only commented on the first.

Comment (mdecleir) [reply DAT_files/hd183143.dat]: Oops, removed those as well!

**Review by karllark — APPROVED** [review]
Looks great.

**Pull request merged**